src/widgets/media-requests/MediaRequestListTile.tsx

```diff
@@ -170,14 +171,19 @@ function MediaRequestListTile({ widget }: MediaRequestListWidgetProps) {
               </Flex>
               <Stack justify="center">
                 <Flex gap="xs">
-                  <Image
+                  <Avatar
                     src={item.userProfilePicture}
-                    width={25}
-                    height={25}
+                    size={25}
                     alt="requester avatar"
                     radius="xl"
-                    withPlaceholder
-                  />
+                  >
+                    <Image
+                      src={item.fallbackUserProfilePicture}
+                      alt="requester avatar"
+                      radius="xl"
+                      withPlaceholder
```

Comment:
Those two are now unnecessary since it should be handled by the Avatar wrapper.
(Just like you did for the MediaRequestStatsTile)